### PR TITLE
data: fatemaker254

### PR DESCRIPTION
**Pull request by fatemaker254**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5320"></a>

### CODE REVIEW

File: data/fatemaker254.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "bio": "An open source enthusiast ",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Sohamdas07",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "www.linkedin.com/in/soham-das07",
```
Comment (imbeshat) [inline data/fatemaker254.json:13]:
```suggestion
      "url": "https://www.linkedin.com/in/soham-das07",
```

File: data/fatemaker254.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/fatemaker254",
+      "icon": "FaGithub"
+    },
```
Comment (imbeshat) [inline data/fatemaker254.json:20]:
```suggestion
    }
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by fatemaker254** [comment]
Thanks a lot everyone for your effort

**Pull request merged**